sporkfish/lichess_bot/lichess_bot_berserk.py

```diff
@@ -144,13 +145,39 @@ def set_pos_and_play_move(
                 num_moves = len(state["moves"].split())
                 prev_moves = state["moves"]
                 set_pos_and_play_move(num_moves, color, prev_moves, game_id, state)
+            elif state["type"] == "opponentGone" and state["gone"]:
```

Comment:
Curious why we need to check both?

Reply:
i'm not too sure why either. https://lichess.org/api#tag/Bot/operation/botGameStream. the api documentation has examples on opponentGoneTrue and opponentGoneFalse, so im checking both just to make sure

Reply:
as in im not sure why it has both opponentgonetrue and opponentgonefalse since it doesnt make sense. maybe it is to detect if the opponent does come back?

Reply:
Yes looks like it. You can recheck that flag after we wait to claim win as a basic solution I suppose